### ISSUE TITLE
Use PSFgen version 2.0

### ISSUE THREAD

**Issue by DomML** [Eigenstate/vmd-python#39]
Hello,
PSFGEN 2.0 from the last VMD 1.9.4 alpha (october 2021) include some new functions (like HMASSREPART). Is there any way to use it with my current install of VMD-python (3.0.6) ?

**Comment by Eigenstate** [comment]
Good point-- it's been a while since I've synchronized this repo with VMD's master branch. I'll get to that in the next few days.

**Comment by Im-2012** [comment]
Hi, I have the this demand, too
